### PR TITLE
 Bug 2091944: crio: Use absolute path for binaries in crio hook script

### PR DESCRIPTION
**Pull request by yanirq**
Fixes Bug 2091944: low-latency-hooks.sh script requires ip binary in container image

The binaries in the script used by cri-o prestart hook are fixed to use absolute path since
$PATH is not guaranteed to contain all the needed paths for binaries invocations in prestart phase.

### CODE REVIEW

File: build/assets/scripts/low-latency-hooks.sh [hunk 1]
```diff
@@ -1,35 +1,38 @@
 #!/usr/bin/env bash
 
+JQ="/usr/bin/jq"
+IP="/usr/sbin/ip"
```
Comment (Tal-or) [inline build/assets/scripts/low-latency-hooks.sh:4]: dito

File: build/assets/scripts/low-latency-hooks.sh [hunk 2]
```diff
@@ -1,35 +1,38 @@
...
+
 mask="${1}"
 [ -n "${mask}" ] || { logger "${0}: The rps-mask parameter is missing" ; exit 0; }
 
-pid=$(jq '.pid' /dev/stdin 2>&1)
+pid=$(${JQ} '.pid' /dev/stdin 2>&1)
 [[ $? -eq 0 && -n "${pid}" ]] || { logger "${0}: Failed to extract the pid: ${pid}"; exit 0; }
 
-ns=$(ip netns identify "${pid}" 2>&1)
+ns=$(${IP} netns identify "${pid}" 2>&1)
```
Comment (Tal-or) [inline build/assets/scripts/low-latency-hooks.sh:12]: Let's use double quotes here for both consistency and safeness: https://tldp.org/LDP/abs/html/quotingvar.html

File: build/assets/scripts/low-latency-hooks.sh [hunk 3]
```diff
@@ -1,35 +1,38 @@
 #!/usr/bin/env bash
 
+JQ="/usr/bin/jq"
```
[the remark this answers is not in the document]
Comment (yanirq) [reply build/assets/scripts/low-latency-hooks.sh:3]: I would rather do the opposite there. We want vars to be capital

[the remark this answers is not in the document]
Comment (fromanirh) [reply build/assets/scripts/low-latency-hooks.sh:3]: Agreed. There is no gold standard here, with https://google.github.io/styleguide/shellguide.html#s7.2-variable-names possibly being the closest, but the general consensus is to use `UPPERCASE_VARS` for constants and env vars (and these fit more as constant than everything else) and `lowercase_vars` for local/function variable

**Pull request merged**